mws/apis/easyship.py

```diff
@@ -0,0 +1,90 @@
+"""
+Amazon Easyship API
+"""
+from __future__ import absolute_import
+
+from ..mws import MWS
+from .. import utils
+
+
+class EasyShip(MWS):
+    """
+        Amazon EasyShip API
+        docs: http://docs.developer.amazonservices.com/en_IN/easy_ship/EasyShip_Overview.html
+    """
+    URI = "/EasyShip/2018-09-01"
+    VERSION = "2018-09-01"
+    NAMESPACE = '{https://mws.amazonservices.in/EasyShip/2018-09-01}'
+
+    def get_pickup_slots(self, marketplace_id=None, amazon_order_id=None, package_width=0,
+                         package_height=0, package_length=0, package_dimensions_uom='cm',
+                         package_weight=0, package_weight_uom='g'):
```

Comment:
Method names throughout the package should match the MWS action name, converted to `lowercase_with_underscores`. Therefore, this method should be changed to `list_pickup_slots`.